tests/Microsoft.DotNet.Framework.Docker.Tests/DockerHelper.cs

```diff
@@ -72,14 +76,56 @@ public static bool ImageExists(string tag)
             return process.ExitCode == 0 && stdOutput != "";
         }
 
-        public void Run(string image, string containerName, string command)
+        public void Run(string image, string containerName, string command, bool web = false)
```

Comment:
The intention of this class is to only contain Docker concepts.  The `web` parameter doesn't fit well as it is tied to the particular test usage.  I think this should be renamed to the Docker argument it controls, `detach`.

Reply:
I was about this change this, I agree, will update this in the next update.